ecl/eclagent/eclagent.cpp

```diff
@@ -3153,7 +3153,7 @@ void EclAgent::reportProgress(const char *progress, unsigned flags)
         CriticalBlock block(wusect);
         WorkunitUpdate w = updateWorkUnit();
         w->setState(WUStateAborting);
-        throw new WorkflowException(0, "Workunit abort request received", (workflow ? workflow->queryCurrentWfid() : 0), WorkflowException::ABORT, MSGAUD_user);
```

Comment:
Is 0 the best we can do here? And is it good enough?

Reply:
This only affects tracing when a query is aborted and will be improved in the next pull request

Reply:
ok